packages/@react-spectrum/provider/stories/Provider.stories.tsx

```diff
@@ -51,6 +51,17 @@ storiesOf('Provider', module)
       </Provider>
     )
   )
+  .add(
```

Comment:
Do we need this storybook example? This exists in the provider docs.

Reply:
yes please. We use it for testing.